src/App.tsx

```diff
@@ -1,24 +1,73 @@
 import React from 'react';
+import { Clock } from './components/Clock';
 import './App.scss';
+/* eslint-disable no-console */
 
-const App: React.FC = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  clockName: number;
+  isClockVisible: boolean;
 };
 
+class App extends React.Component<{}, State> {
+  state = {
+    clockName: 0,
+    isClockVisible: true,
+  };
+
+  showClock = () => {
```

Comment:
I would suggest reducing the code and create one method instead of these two: hideClock and showClock.